packages/connect-node/src/private/node-request.ts

```diff
@@ -47,11 +45,11 @@ export function makeNodeRequest(options: NodeRequestOptions) {
     const endpoint = new URL(options.req.url);
     const nodeRequestFn = nodeRequest(endpoint.protocol);
     const request = nodeRequestFn(options.req.url, {
+      ...options.httpOptions,
```

Comment:
This is crucial. While we omit "signal" from the type, it is still possible to pass options with a "signal" property to this. So we have to make sure we override it.